www/app/login/login.controller.js

```diff
@@ -39,18 +49,21 @@
         password === userInfo.password;
     }
 
-    function authenticate(username, password) {
-
-      LoginFirebaseService.getUserInfo(username)
-        .then(function(userInfo) {
-          if (_credentialsAreCorrect(username, userInfo, password)) {
-            _authSuccess();
-            RevisionService.setUsername(username);
-          }else {
-            authError();
-          }
-        });
+    function authenticate(username, password){
+      if(areEmptyCredentials(username, password)){
+        LoginFirebaseService.logIn(username, password)
+            .then(handleCredentials);
```

Comment:
```
.then(_authSuccess)
.catch(authError)
```

rather above's approach instead of checking is the response is instance of `Error`. Also keep consistency on method naming: `_authSuccess` vs `authError`